HdProduction.App.Common/Auth/AuthenticationExtensions.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.AspNetCore.Authentication;
```

Comment:
This file isn't related to MQ removing?

Reply:
it is related to jwt auth, not to mq removing